### ISSUE TITLE
Account recovery request

### ISSUE THREAD

**Issue by nisaac21** [pypi/support#3230]
### PyPI Username

`nisaac21`

### Reason for request

Lost security token, computer data was wiped 

### Code of Conduct

- [X] I agree to follow the PSF Code of Conduct

### Acknowledgement

- [X] I understand that it may take a significant amount of time to process my account recovery request.

**Comment by ewdurbin** [comment]
User `nisaac21` has valid recovery codes generated as of 2023-09-01 19:40:54.728527.

Please use those recovery codes to regain access to your account.

**Comment by nisaac21** [comment]
Where do I access those recovery codes?

**Comment by nisaac21** [comment]
I need new ones as I do not have access to those recovery codes

**Comment by ewdurbin** [comment]
You were instructed to download or copy the codes to a secure place when enrolling in 2FA. Please reference that location for your codes.

**Comment by nisaac21** [comment]
Yes they were stored securely  locally but my drive got wiped due to a computer malfunction and I lost access. Is there any way to recover my account or have those codes resent or regenerated? I have no access to the 2FA of any form but have the email account access and this github account access - both of which are linked to my PyPi account

**Comment by nisaac21** [comment]
If not possible, can I have the account deleted and the package removed so I can restart and re-upload everything? Is there any work around in this situation?

**Comment by ewdurbin** [comment]
Contacted via primary email address on the account, please read the instructions and reply to proceed with recovery.

**Comment by Thespi-Brain** [comment]
Hi @nisaac21, 

I am following up regarding your request. Are you still having an issue regarding your recovery codes and logging in? Let us know! I will keep this issue open for another week unless I hear from you earlier. After that, if you are still having trouble, feel free to create a new request. Thanks!

**Comment by Thespi-Brain** [comment]
Closing due to non response.
